gbdxtools/images/tms_image.py

```diff
@@ -77,13 +77,10 @@ def raise_aoi_required():
 
 
 class TmsMeta(object):
-    def __init__(self, access_token=os.environ.get("MAPBOX_API_KEY"),
-                 url="https://api.mapbox.com/v4/mapbox.satellite/{z}/{x}/{y}.png",
-                 zoom=22, bounds=None):
+    def __init__(self, url, zoom=18, bounds=None):
         self.zoom_level = zoom
-        self._token = access_token
         self._name = "image-{}".format(str(uuid.uuid4()))
-        self._url_template = url + "?access_token={token}"
```

Comment:
Since we're no longer doing templating let's get rid of `._url_template` and just make it `._url`